package-testing/python-sdk-relay/build-and-run.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+# Set default values for vars
+
+: "${SDK_REF:=main}"
+: "${SDK_RELAY_HOST:=localhost}"
+: "${SDK_RELAY_PORT:=4000}"
+SDK="https://github.com/Eppo-exp/eppo-multiplatform.git"
+
+# checkout the specified ref of the SDK repo, build it, and then insert it into vendors here.
+rm -rf tmp
+mkdir -p tmp
+
+echo "Cloning ${SDK}@${SDK_REF}"
+git clone -b ${SDK_REF} --depth 1 --single-branch ${SDK} tmp || ( echo "Cloning repo failed"; exit 1 )
+
+# Run the poller
+python3 -m venv tmp/.venv
+source tmp/.venv/bin/activate
+pip install maturin
+
+pip install -r requirements.txt
+
+# Build the wheel file in tmp directory
+maturin build --release --out tmp/dist --find-interpreter --manifest-path ./tmp/python-sdk/Cargo.toml
```

Comment:
makes more sense for me to have this compile command in the repo

Reply:
Agree. The build process might change *and* it needs to be in sync with the changes in multiplatform repo.

Furthermore, having the build here circumvent our regular build/release defeats the main purpose of these tests — verifying that packaging is correct.

Ideally, we want to separate the build/packaging/release from testing completely. The closest we could get is either using pre-releases or using a private package repository.